### PR TITLE
Peridaxon healing is now properly clamped.

### PR DESCRIPTION
**Pull request by MadmanMartian**
Hopefully no more of this negative lung damage

Not yet tested. Will test in the morning.

### CODE REVIEW

**Comment by PJB3005** [comment]
@DamianX why the needs fixing?

**Comment by DamianX** [comment]
Oh shit, didn't post the review

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 1]
```diff
@@ -2595,7 +2595,7 @@
 		var/datum/organ/external/chest/C = H.get_organ(LIMB_CHEST)
 		for(var/datum/organ/internal/I in C.internal_organs)
 			if(I.damage > 0)
-				I.damage -= 0.20
+				I.damage = max(0,I.damage-=0.20)
```
Comment (PJB3005) [inline code/modules/reagents/Chemistry-Reagents.dm]: You're doing `max(0,I.damage-=0.20)`, you probably want to do `max(0, I.damage - 0.20)`. Note the accidental `-=`

**Review by PJB3005 — DISMISSED** [review]

File: code/modules/reagents/Chemistry-Reagents.dm (hunk 1 above)
Comment (DamianX) [inline code/modules/reagents/Chemistry-Reagents.dm]: should just be `max(0, I.damage - 0.20)`, no need to use the assignment operator in there

Comment (MadmanMartian) [reply code/modules/reagents/Chemistry-Reagents.dm]: thanks for catching that. Coding at 3AM with a christmas headache is not recommendable to anyone.

Comment (MadmanMartian) [reply code/modules/reagents/Chemistry-Reagents.dm]: thanks for catching that. Coding at 3AM with a christmas headache is not recommendable to anyone.

**Review by PJB3005 — APPROVED** [review]

**Pull request merged**